pkg/morph/client/client.go

```diff
@@ -235,6 +235,11 @@ func (c *Client) TestInvokeIterator(contract util.Uint160, method string, args .
 		items = append(items, ii...)
 	}
 
+	err = c.rpcActor.TerminateSession(sid)
+	if err != nil {
+		return nil, fmt.Errorf("session (%s) termination: %w", sid, err)
+	}
+
```

Comment:
And one more adjustment: let's skip error check here. It's not important for the client, and current version of NeoGo will return an error if session was already terminated.

Reply:
sure. via `defer` then